### ISSUE TITLE
Charliecloud can't pull biocontainers images

### ISSUE THREAD

**Issue by Mia0509** [nf-core/modules#3602]
### Have you checked the docs?

- [X] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
- [X] [nf-core modules documentation](https://nf-co.re/docs/contributing/modules)

### Description of the bug

While trying to use a nf-core pipeline on a server which uses charliecloud, i encountered this error : 

```
ERROR ~ Error executing process > 'NFCORE_HIC:HIC:INPUT_CHECK:SAMPLESHEET_CHECK'

Caused by:
  Charliecloud failed to pull image
  command: ch-image pull -s /Xnfs/abc/charliecloud biocontainers/python:3.8.3 > /dev/null
  status : 1
  message:
    pulling image:    biocontainers/python:3.8.3
    requesting arch:  amd64
    error: unauthorized or not in registry: registry-1.docker.io:443/biocontainers/python:3.8.3
    hint: if your registry needs authentication, use --auth



 -- Check '.nextflow.log' file for details
```
I assume it is due to the commit #3380 which remove the prefix `quay.io/` from almost every biocontainer and uses `docker.registry` to give this prefix in the nextflow config file.

Is there an equivalent to `docker.registry` that I could use to fix this issue? 

Thank you for your help

**Comment by bozmik** [comment]
Check the solution from this issue: https://github.com/nf-core/modules/issues/3589. 
Adding the docker.registry = 'quay.io' to the config file solved the problem for me.

**Comment by Mia0509** [comment]
> Check the solution from this issue: #3589. Adding the docker.registry = 'quay.io' to the config file solved the problem for me.

Yes, it works when you use docker to run your pipeline or modules
But I'm using Charliecloud, and according to the documentation, there's no equivalent of `docker.registry` for Charlicloud...

**Comment by luisas** [comment]
This should be fixed by using `charliecloud.registry = 'quay.io'` in your config. 

This is also now part of the standard nextflow.config in nf-core pipelines --> e.g. https://github.com/nf-core/multiplesequencealign/blob/de24fb11e5ff385193c77bd1a0e2bcc9922975ac/nextflow.config#L233C1-L233C34
